README.md

```diff
@@ -78,6 +81,22 @@ The domain should be a Guardian-owned domain. For internal tools,
 Name of the artifact containing the static resources. Should be uploaded in
 an earlier workflow step.
 
+## How to decommission a static site that is no longer needed
+
+1. **Remove the stack**
+   You need access to the deployTools account to do this. Either get the permissions from janus, or ask DevX.
+   Delete the `deploy-PROD-<app>` stack.
```

Comment:
minor suggestion to detail [what resources are being deleted](https://github.com/guardian/actions-static-site/blob/main/cdk/static-site.ts):

```suggestion
   Delete the `deploy-PROD-<app>` stack. This will remove the resources defined in [static-site.ts](./cdk/static-site.ts)
```